prometheus/README.md

```diff
@@ -1,10 +1,27 @@
 # Export metrics to Prometheus
 
-Kafka uses JMX to expose metrics, as is already [enabled](https://github.com/Yolean/kubernetes-kafka/pull/96) for broker pods. There's many ways to use JMX. For example [Kafka Manager](../yahoo-kafka-manager/) uses it to display current broker traffic.
+JMX is already [enabled](https://github.com/Yolean/kubernetes-kafka/pull/96) for broker pods (TODO extract to kustomization). There's many ways to use JMX. For example [Kafka Manager](../yahoo-kafka-manager/) uses it to display current broker traffic.
 
-At Yolean we use Prometheus. This folder adds a sidecar to the broker pods that exports selected JMX metrics over HTTP in Prometheus format. To add a container to an existing pod we must use the `patch`command:
+This folder adds a sidecar to the broker pods that exports selected JMX metrics over HTTP in Prometheus format. To add a container to an existing pod we must use the `patch`command:
+
+Using kubectl 1.14+
+
+```
+kubectl --namespace kafka apply -k prometheus/
+```
+
+Using pre-1.14 kubectl:
 
 ```
 kubectl --namespace kafka apply -f prometheus/10-metrics-config.yml 
 kubectl --namespace kafka patch statefulset kafka --patch "$(cat prometheus/50-kafka-jmx-exporter-patch.yml )"
 ```
+
+## Consumer lag monitoring
+
+See [Burrow](../linkedin-burrow)
+or [Kafka Minion](../consumers-prometheus/)
```

Comment:
Maybe some additional comments what one may prefer depending on the use case / environment?

- Many kafka clusters to monitor with just one Exporter? => Burrow
- Only interested in Consumer **Health check**? => Burrow
- Want metrics in prometheus? => Kafka Minion
- Looking for HA support? => Kafka Minion
- Using versioning in group ids (e. g. consumer group name "email-sender-5" where 5 indicates the version) ? => Kafka Minion

In fact they can supplement each other and it may be a valid desire to operate both of them.

Reply:
There's lots and lots of research to be done for anyone who wants to set up a Kafka stack and I see this repository as a collection of examples rather than a way to discuss the choices.